framework/src/source/JUnitTestReporter.bs

```diff
@@ -1,11 +1,7 @@
 namespace rooibos
   class JUnitTestReporter extends rooibos.BaseTestReporter
 
-    function new(testRunner as dynamic)
-      super(testRunner)
-    end function
-
-    override function reportResults(allStats as dynamic)
+    override function onEnd(allStats as dynamic)
```

Comment:
```suggestion
    override function onEnd(event as object)
```

Reply:
Done. I went one step further and declared an interface `ITestReporterOnEndEvent`. Let me know if that's overkill and I can scale it back.